IfcMechanicalFastener Rail Import BIMQ 2021-07-21T07:45:27.4452357Z (#189)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/i/IsLiftingBracketConnection_0b7QU$LAbEywG5aCANU6MN/Documentation.md

> Indicates if it's a connection between two different heights (TRUE) or not (FALSE).

Reviewer: ```suggestion
Indicates if the connection is between two different heights (TRUE) or not (FALSE).
```
